presto-docs/src/main/sphinx/release/release-0.117.rst

```diff
@@ -0,0 +1,10 @@
+=============
+Release 0.117
+=============
+
+General Changes
+---------------
+
+* Add cast between JSON and VARCHAR back so that the casts and functions
```

Comment:
I would say "Add back casts between JSON and VARCHAR to provide an easier migration path to :func:`json_parse` and :func:`json_format`. These will be removed in a future release."